files/translations/content/bee_movie.csv

```diff
@@ -0,0 +1,2 @@
+_,en,ru,pt-br,es-es,de,fr-fr,it,pl,zh-cn,jp,ko,,NOTES – use \n for newline,max length,,,,,,,,,,
```

Comment:
What's with the morse/binary at the end? `langmix` won't ever swap to that since its out of range, in case you thought that.

Just English/French should be fine I think

Reply:
ooooooooooooooo fuck, ive been extending stuff beyond the range of langmix this entire time to make space for actual TLs, can langmix be changed to exclude the problematic languages instead of just having a hard range?